sympy/integrals/meijerint.py

```diff
@@ -1016,7 +1050,10 @@ def _c1():
         def _cond(z):
             '''Returns True if abs(arg(1-z)) < pi, avoiding arg(0).
 
-            Note: if `z` is 1 then arg is NaN. This raises a
+            Explanation
+            ===========
+
+            If `z` is 1 then arg is NaN. This raises a
             TypeError on `NaN < pi`. Previously this gave `False` so
```

Comment:
The single backticks in this docstring should be updated to use double backticks.

Reply:
Done.